StopwatchExercise/Watch.swift

```diff
@@ -6,3 +6,48 @@
 //
 
 import Foundation
+import Combine
+
+class Watch: ObservableObject {
+    @Published var isStarted: Bool = false
+    @Published var remainingTime: Int = 1500
+    @Published var time: String = "25:00"
+    
+    private var timerPublisher = Timer.publish(every: 1.0, on: .main, in: .common).autoconnect().eraseToAnyPublisher()
+    private var cancellable: AnyCancellable? = .none
+    
+    init(timerPublisher: AnyPublisher<Publishers.Autoconnect<Timer.TimerPublisher>.Output, Publishers.Autoconnect<Timer.TimerPublisher>.Failure>) {
```

Comment:
```suggestion
    init(timerPublisher: AnyPublisher<Date, Never>) {
```

On devrait pouvoir mettre le type du publisher sous cette forme. C'est plus clair.